### PR TITLE
Add workflows and scripts, bump symfony/console, require nextcloud/co…

### PR DESCRIPTION
**Pull request by come-nc**
…ding-standard

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

**Comment by come-nc** [comment]
Is the vendor directory versioned on purpose? Is it ok to bump symfony/console dep like I did?

**Comment by kesselb** [comment]
> Is the vendor directory versioned on purpose?

I don't know. Maybe related to the build script for the phar file.

> Is it ok to bump symfony/console dep like I did?

Looks good to me. Some dev dependencies are new. Could we scope them like https://github.com/nextcloud/server/pull/28507 to not add more dependencies to git?

**Comment by come-nc** [comment]
Looking into the updater.phar that I get after a `make updater.phar` I think we are fine because the namespaces listed in the phar do not include any dev dependency.
The new updater.phar is 30K larger than master, but this is consistent with the dependency bump of symfony/console which is also larger.

**Comment by come-nc** [comment]
Pro tip: `./box info -l updater.phar` can be used to list content and sizes of the updater, and then diff that with master to see what actually changed in the phar. Changes seems to be correct.

**Comment by come-nc** [comment]
I’m cancelling trying to update the tested version numbers to supported versions because the signatures are hardcoded and it’s getting too complicated. So for now we’ll keep testing the updater with 19-21 and master.

**Comment by come-nc** [comment]
Moved the testing from drone to github workflow to be able to setup any PHP version, everything is green again, except master but it should turn green tomorrow thanks to https://github.com/nextcloud-gmbh/release-script/pull/58
I need someone with admin access to remove the drone job from expected jobs.

Apart from that and a bit of squashing I think we are good to merge.

I will open other PR to maybe add psalm and in anycase fix the 8.1 warnings.

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+use Nextcloud\CodingStandard\Config;
+
+$config = new Config();
+$config
+	->getFinder()
+	->notPath('build')
+	->notPath('l10n')
+	->notPath('src')
+	->notPath('vendor')
```
Comment (kesselb) [inline .php-cs-fixer.dist.php:15]: Would be nice to exclude the new `vendor-bin` directory too.

**Review by kesselb — APPROVED** [review]
LGTM :+1:

**Comment by blizzz** [comment]
do we need backports?

**Pull request merged**